shared/js/initApp.js

```diff
@@ -29,7 +29,7 @@ window.apps = {
       cdoSounds: CDOSounds,
       position: {blockYCoordinateInterval: 25},
       onInitialize: function() {
-        dashboard.createCallouts(this.callouts);
+        dashboard.createCallouts(this.level.callouts || this.callouts);
```

Comment:
Looks like either should work (I used `this` because it was already doing that for `this.callouts`):
L162, the app's main is called with appOptions:
`    window[appOptions.app + 'Main'](appOptions);`
Each of our apps calls `appMain` with that param as the 3rd param:
`      appMain(window.Studio, levels, options);`
L72 of appMain.js shows that we invoke onInitialize from the options object (making it `this` inside):
`        options.onInitialize();`